doc.go

```diff
@@ -169,6 +169,11 @@ Note: GOSNOWFLAKE_SKIP_REGISTERATION should not be used if sql.Open() is used as
 to connect to the server, as sql.Open will require registration so it can map the driver name
 to the driver type, which in this case is "snowflake" and SnowflakeDriver{}.
 
+After Version 1.11.2 and later, you can load the connnection configuration with .toml file format.
```

Comment:
You don't have to specify version - this file is versioned with the code.

Reply:
I removed the version.